lib/manager/conan/extract.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { loadFixture } from '../../../test/util';
+import { extractPackageFile } from '.';
+
+const conanfile1 = loadFixture('conanfile.txt');
+const conanfile2 = loadFixture('conanfile2.txt');
+const conanfile3 = loadFixture('conanfile.py');
+
+describe('manager/conan/extract', () => {
+  describe('extractPackageFile', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('nothing here')).toBeNull();
+    });
+    it('extracts multiple image lines from conanfile.txt', () => {
+      const res = extractPackageFile(conanfile1);
+      expect(res?.deps).toHaveLength(10);
```

Comment:
Can you please add some more explicit expects please?

https://codewithhugo.com/jest-array-object-match-contain/